core-db/src/main/java/io/novafoundation/nova/core_db/migrations/42_43_StakingRewardPeriods.kt

```diff
@@ -0,0 +1,28 @@
+package io.novafoundation.nova.core_db.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+val StakingRewardPeriods_42_43 = object : Migration(42, 43) {
+
+    override fun migrate(database: SupportSQLiteDatabase) {
+        createCoinPriceTable(database)
+    }
+
+    private fun createCoinPriceTable(database: SupportSQLiteDatabase) {
+        database.execSQL(
+            """
+            CREATE TABLE IF NOT EXISTS `staking_reward_period` (
+            `accountId` BLOB NOT NULL, 
+            `chainId` TEXT NOT NULL, 
+            `assetId` INTEGER NOT NULL, 
+            `stakingType` TEXT NOT NULL, 
+            `periodType` TEXT NOT NULL, 
+            `customPeriodStart` INTEGER, 
+            `customPeriodEnd` INTEGER, 
+            `offsetFromCurrentDate` INTEGER, 
```

Comment:
Why do we need to store offset from current date?

Reply:
I use it to save period in time (week, month, quarter etc.)